<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCefViewTestClass</class>
 <widget class="QMainWindow" name="QCefViewTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>824</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCefViewTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">#centralWidget {
background-color: pink;
}</string>
   </property>
   <widget class="QWidget" name="cefContainer" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>1280</width>
      <height>768</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#cefContainer {
background-color: white;
}</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_cefContainer">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEdit_url">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>951</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_forward">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Forward</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stop">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reload">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reload</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qcefviewtest.qrc"/>
 </resources>
 <connections/>
</ui>
